go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/cometbft/cometbft-db
 
-go 1.21
+go 1.20
```

Comment:
?

Reply:
Will revert, it was left over after testing something for the validator experiments.